contrib/mesos/pkg/scheduler/podtask/pod_task.go

```diff
@@ -33,8 +33,8 @@ import (
 )
 
 const (
-	DefaultContainerCpus = 0.25 // initial CPU allocated for executor
-	DefaultContainerMem  = 64   // initial MB of memory allocated for executor
+	DefaultContainerCpus = 0.25                   // initial CPU allocated for executor
+	DefaultContainerMem  = 64.0 * 1024.0 * 1024.0 // initial MB of memory allocated for executor
```

Comment:
k8s has a nice "resource" package that defines a Quantity type: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/resource/quantity.go

perhaps this could help to streamline things a bit?